docs/swagger.yaml

```diff
@@ -0,0 +1,216 @@
+basePath: /
```

Comment:
question: is this an autogenerated Swagger spec?

Reply:
Unfortunately no, and it should not be there... this is some experiment with LightHouse and unrelated to this PR.